assets/js/app.js

```diff
@@ -91,19 +91,35 @@ const listeners = {
     if (relativeElementRect.width < e.target.offsetWidth) { e.target.style.left = 0 }
   },
   "chat:select-message": (e) => {
-    setTimeout(() => {
-      if (document.querySelector("#chat-messages").classList.contains('selectMode') == false) { return false }
+    const messageBlock = e.target;
+    const messageBlockCheckbox = messageBlock.querySelector('.selectCheckbox');
+    messageBlock.classList.toggle('selectedMessageBackground');
+    messageBlockCheckbox.classList.toggle('checked');
+    messageBlockCheckbox.checked = !messageBlockCheckbox.checked;
 
-      const messageBlock = e.target;
-      const messageBlockCheckbox = messageBlock.querySelector('.selectCheckbox');
-      messageBlock.classList.toggle('selectedMessageBackground');
-      messageBlockCheckbox.classList.toggle('checked');
-      messageBlockCheckbox.checked = !messageBlockCheckbox.checked;
+    setTimeout(() => {
+      if (document.querySelector("#chat-messages").classList.contains('selectMode') == false) {return false }
+     
+      const allCheckboxes = document.querySelectorAll('.checked')
+      
       if (document.querySelectorAll('.checked').length == 0) {
         document.getElementById("chatContent").dispatchEvent(
           new CustomEvent('chat:toggle-selection-mode', { detail: { chatType: e.detail.chatType } })
         )
       }
+      const deleteButton = document.getElementById("delete-btn");
```

Comment:
this functionality should work for rooms too
are we sure that we have only one #delete-btn?